### PR TITLE
Added Profile & Posts Page

### PR DESCRIPTION
**Pull request by el634dev**
Issues #12 , #16 , #17

### CODE REVIEW

File: templates/browse.html [hunk 1]
```diff
@@ -40,36 +44,120 @@
...
-        Some quick example text to build on the card title and make up the bulk
-        of the card's content.
-      </p>
-    </div>
-    <!--Buttons-->
-    <div class="card-body">
-      <a href="#" class="card-link">Share</a>
-      <a href="#" class="card-link">Download</a>
-    </div>
+<div class="container container-fluid embed-responsive embed-responsive-16by9 mt-3">
```
Comment (b3fr4nk) [inline templates/browse.html:47]: Visually looks really good but none of this uses the backend routes I created. It should get the posts already created and display them. Also you are missing the post details page but that ticket has been moved to finished.

File: templates/browse.html [hunk 2]
```diff
@@ -15,9 +17,11 @@
             Browse
           </a>
         </li>
-        <!--About Link-->
+        <!--Posts Link-->
```
Comment (b3fr4nk) [inline templates/browse.html:20]: Posts leads to the same place as Browse. Should maybe be new post and lead to the new post page

File: templates/post.html [hunk 3]
```diff
@@ -39,6 +43,99 @@
   </div>
 </nav>
 <!--End of Navbar-->
+<!--Comment Section-->
+<div class="container container-fluid mt-5 mb-5">
+  <div class="row height d-flex justify-content-center align-items-center">
```
Comment (b3fr4nk) [inline templates/post.html:48]: Identical to the browse page?

File: templates/profile_page.html [hunk 4]
```diff
@@ -0,0 +1,397 @@
+{% extends 'base.html' %}
```
Comment (b3fr4nk) [inline templates/profile_page.html:1]: How do you get here?

File: templates/signin.html [hunk 5]
```diff
@@ -126,13 +126,13 @@ <h5>Support</h5>
           FAQ
         </a>
       </li>
-      <!--Browse-->
+      <!--Resources-->
```
Comment (b3fr4nk) [inline templates/signin.html:129]: Header and footer should probably use a template so we only need to make the change once

File: templates/new_post.html [hunk 6]
```diff
@@ -15,9 +15,9 @@
             Browse
           </a>
         </li>
-        <!--About Link-->
+        <!--Posts Link-->
```
Comment (b3fr4nk) [inline templates/new_post.html:18]: No way for the user to get here

File: templates/browse.html [hunk 7]
```diff
@@ -101,13 +187,13 @@ <h5>Quick Links</h5>
       </li>
       <!--Browse-->
       <li class="nav-item mb-2">
-        <a href="#" class="nav-link p-0 text-muted text-decoration-none">
+        <a  href="{{ url_for('pages.browse_page') }}" class="nav-link p-0 text-muted text-decoration-none">
```
Comment (b3fr4nk) [inline templates/browse.html:85]: there is a full backend route for this. the get all posts

File: templates/signup.html [hunk 8]
```diff
@@ -82,13 +80,13 @@ <h5>Quick Links</h5>
       </li>
       <!--Browse-->
       <li class="nav-item mb-2">
-        <a href="#" class="nav-link p-0 text-muted text-decoration-none">
+        <a href="{{ url_for('pages.browse_page') }}"  class="nav-link p-0 text-muted text-decoration-none">
```
Comment (b3fr4nk) [inline templates/signup.html:83]: again a proper route for this

**Review by b3fr4nk — CHANGES_REQUESTED** [review]
There seems to be a lot of miscommunication I think we need to meet

**Comment by el634dev** [comment]
Creating new pr

**Pull request closed without merging**